README.md

```diff
@@ -40,7 +40,7 @@ Give a ⭐️ if you like this project!!
 
 - [Microverse](https://microverse.org)
 - Inspiration
-- etc
+- etc.
 
 ## 📝 License
 
```

Comment:
- [ ] I have reviewed your past commits to this project and I have added all necessary comments